projects/api-twitter/server/utils/send-email.js

```diff
@@ -0,0 +1,31 @@
+// 1. importar la librería - npm install nodemailer
+const nodemailer = require("nodemailer");
+const sgTransport = require("nodemailer-sendgrid");
+const config = require("../config");
+const {
+  welcomeHtmlTemplate,
+  welcomeTextTemplate,
+} = require("./email-template");
+
+async function sendWelcomeEmail(username, email) {
+  const options = {
+    apiKey: config.email.sengridApiKey,
+  };
+
+  // 2. Crear el transport (Fake - Ethereal, Gmail, Sendgrid, MailGun, Postmark)
+  let transporter = nodemailer.createTransport(sgTransport(options));
+
+  // 3. Send Email
+  await transporter.sendMail({
+    from: `"${config.email.senderEmailUsername}" <${config.email.senderEmail}>`, // sender address
+    to: `${email}`, // list of receivers
+    subject: `Bienvenido ${username}`, // Subject line
+    text: welcomeTextTemplate({ name: `${username}` }), // plain text body
+    html: welcomeHtmlTemplate({ name: `${username}` }), // html body
+  });
+
+  console.log("Correo enviado");
```

Comment:
estos mensajes son muy comunes en cualquier desarrollo back para saber el estado de las operaciones que se ejecutan en el back te aconsejaría que formalizaras un poco la estructura de mensaje en el `console.log` ej: `[SUCCESS]: mail message sended to user ${user_id}`.

Reply:
listo